packages/terra-toggle-section-header/src/terra-dev-site/test/toggle-section-header/ProgrammaticToggle.test.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from 'terra-button';
+import ToggleSectionHeader from '../../../ToggleSectionHeader';
+
+class ProgrammaticToggle extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false,
+    };
+    this.toggle = this.toggle.bind(this);
+  }
+
+  toggle() {
+    this.setState(prevState => ({ isOpen: !prevState.isOpen }));
+  }
+
+  render() {
+    return (
+      <div>
+        <Button id="toggle" text="Trigger Toggle" onClick={this.toggle} />
+        <ToggleSectionHeader isOpen={this.state.isOpen} title="Default Toggle Section Header">
```

Comment:
any impact with `isInitallyOpen` is provided?

Reply:
If IsInitiallyOpen is passed in the toggle-section-header is open initially, and the toggle button works accordingly ie it closes on first click. No ill-impacts as such.